src/Network/Kontiki/Raft/Leader.hs

```diff
@@ -107,12 +107,15 @@ quorumIndex = do
     return $ sorted !! (quorum - 1)
 
 -- | Handles `ElectionTimeout'.
-handleElectionTimeout :: (Functor m, Monad m)
+handleElectionTimeout :: (Monad m)
                       => TimeoutHandler ElectionTimeout a Leader m
-handleElectionTimeout = currentState
+handleElectionTimeout =  do
+  resetElectionTimeout
```

Comment:
Similar question as the similar heartbeat timeout change in Candidate or Follower state